app/controllers/lots_controller.rb

```diff
@@ -74,6 +73,7 @@ def lot_params
       :state,
       :price,
       :category_id,
+      :search,
```

Comment:
Not needed.
1. You access search word directly through `params[:search]` and **not** through `lot_params[:search]`.
2. There's no point to permit a param which is not in you model's attributes.